src/main.rs

```diff
@@ -485,6 +497,30 @@ impl AsyncComponent for App {
             gtk::glib::Propagation::Stop
         });
 
+        #[cfg(any(target_os = "macos", target_os = "linux"))]
+        let tray_img = Icon::from_png_bytes(TRAY_ICON).expect("Tray icon exists and is a valid PNG; qe");
+        #[cfg(target_os = "windows")]
+        let tray_img = Icon::from_resource(1, None).expect("Tray icon exists and is a valid PNG; qe");
```

Comment:
I see you've added macOS, are you sure it works there? Because if you're not sure we either need to confirm with someone or not enable this on macOS yet. I think @abhi3700 might be able to test it on macOS.

Also there is still a typo:
```suggestion
        #[cfg(any(target_os = "macos", target_os = "linux"))]
        let tray_img = Icon::from_png_bytes(TRAY_ICON).expect("Tray icon exists and is a valid PNG; qed");
        #[cfg(target_os = "windows")]
        let tray_img = Icon::from_resource(1, None).expect("Tray icon exists and is a valid PNG; qed");
```

Reply:
I  am not confirm it work on macos, but i can confirm without this macos will build fail